python/cog/server/runner.py

```diff
@@ -408,6 +408,10 @@ def _upload_files(self, output: Any) -> Any:
         try:
             # TODO: clean up output files
             return self._file_uploader(output)
+        except (FileNotFoundError, NotADirectoryError):
+            # Errors that indicate an output path does not exist are not considered
+            # infrastructure errors.
```

Comment:
Can we clarify what an "infrastructure error" is and why doing this makes it not one, please?